src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -356,7 +356,7 @@ def _parse_requests(self, requests: dict) -> LLMMetrics:
         """Parse each request in profile data to extract core metrics."""
         raise NotImplementedError
 
-    def get_statistics(self, infer_mode: str, load_level: int | float) -> Statistics:
+    def get_statistics(self, infer_mode: str, load_level: str) -> Statistics:
```

Comment:
Functionally there is no typing in python but we want to do type checking one day.
Is string correct here?

Reply:
Yes. The current args are passed in as string [here](https://github.com/triton-inference-server/client/blob/fcfb475234a958b67598353acdb27aeff32a7ed8/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/parser.py#L58).

If you remove the string formatting there, it will fail with the below:
![image](https://github.com/triton-inference-server/client/assets/58150256/e6554ada-fc6a-4b90-8886-05b89767e3db)

This PR is just making load level optional, so it brings the default load level to parity with the current way we pass them in (as strings). If we wanted to pass them in as ints/floats instead (which is more intuitive, in some ways), we'd need to clean up the flow in places like [parse_profile_data](https://github.com/triton-inference-server/client/blob/fcfb475234a958b67598353acdb27aeff32a7ed8/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/llm_metrics.py#L341) in LLM Metrics. We could probably do that by then converting the args [here](https://github.com/triton-inference-server/client/blob/fcfb475234a958b67598353acdb27aeff32a7ed8/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/llm_metrics.py#L345-L346) to ints/floats. That'd be the implementation, plus some error handling, unless there are other failures to look at. That seems outside of the scope of this PR.

Reply:
You have clearly reasoned this out. 
Thanks for breaking it down for me. 
I am okay with this change and we can revisit/refactor this later if we change our approach.